packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -66,6 +67,27 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
     props.onFetchAvatarPersonaData
   );
 
+  /** we would want to use the onRenderRemoteVideoTile prop in the videoGallery to do something like this only
+   * rendering the transient tile when the remote user is in those states, otherwise we would want to use the defaule behavior
+   */
+  const onRenderTransientVideoTile = (remoteParticipant: VideoGalleryRemoteParticipant): JSX.Element | undefined => {
+    if (remoteParticipant.state !== ('connecting' || 'Ringing')) {
```

Comment:
This is where I think it is preferable to have two separate selectors, one each for connected and transient state participants. So that you don't have this logic about what states are transient etc here.

Reply:
thinking about this I think I agree, because we can provide the other tile based on whether they are a transient user or a connected user